src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -121,6 +121,7 @@ def put(self, request, group, integration_id):
             external_issue.update(**defaults)
 
         try:
+            installation.store_issue_last_defaults(group.project_id, request.DATA)
```

Comment:
can this actually raise an IntegrationFormError? if not, it shouldn't be in this block

Reply:
Yeah this probably doesn't need to be in the try.